python_modules/dagster/dagster/_core/definitions/reconstruct.py

```diff
@@ -434,29 +434,51 @@ def make_bar_job():
             )
         )
 
+    code_pointer = _find_code_pointer(target)
+    if code_pointer:
+        return bootstrap_standalone_recon_pipeline(code_pointer)
+    else:
+        raise DagsterInvariantViolationError(
+            "reconstructable() can not reconstruct jobs or pipelines defined in interactive "
+            "environments like <stdin>, IPython, or Jupyter notebooks. "
+            "Use a pipeline defined in a module or file instead, or use build_reconstructable_job."
```

Comment:
nit and I know it predates but you can probably just get rid of the pipeline verbiage here